src/ucp/proto/proto_select.h

```diff
@@ -24,6 +24,37 @@
 #define UCP_PROTO_SELECT_PARAM_STR_MAX 128
 
 
+typedef struct {
+    ucp_proto_perf_range_t super;
+    size_t                 cfg_thresh; /* Configured protocol threshold */
+} ucp_proto_select_range_t;
+
+
+/**
+ * Protocol and its private configuration
+ */
+typedef struct {
+    /* Protocol definition */
+    const ucp_proto_t        *proto;
+
+    /* Protocol private configuration space */
+    const void               *priv;
+
+    /* Endpoint configuration index this protocol was selected on */
+    ucp_worker_cfg_index_t   ep_cfg_index;
+
+    /* Remote key configuration index this protocol was elected on (can be
```

Comment:
minor: better use `selected` for consistency with `ep_cfg_index` comment

Reply:
fixed